src/main/ngapp/src/app/login/login.component.html

```diff
@@ -44,6 +44,15 @@
       Don't have an account? <a [routerLink]="['/register']">Sign up here</a>
     </div>
     <br />
+
+    <div class="register_link">
+      Forgot Password? 
+      <a routerLink="" (click)="sendPasswordResetEmail(userEmail.value)">
+        Get email to reset
+      </a>
+    </div>
+    <br />
```

Comment:
Can we hide this until a user inputs a wrong password?

Reply:
alright
on it

Reply:
implemented